DisplayBatteryHealth.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include "BMS_BatteryCheck.h"
+
+
+const char *BatteryHealthMessage[] =
+{
+    "is Unstable! Take Action Immediately!!!",
+    "is Stable"
+};
+
+
+/**
+***************************************************************************************************
+* Function Name: PrintBatteryHealthStatus
+* 
+* Function Description: Displays the BMS battery status message.
+*
+* \param  BatteryData_t BatteryData
+*         int IndexOfBatteryHealthMessage
+*         
+* \return  void
+* \retval  None
+***************************************************************************************************
+*/
+void PrintBatteryHealthStatus(BatteryData_t BatteryData, int IndexOfBatteryHealthMessage)

```

Comment:
Nice way to combine the name of the data with the message :)
Can you think of calling this in one place, instead of multiple? We can then change it easily when another form of messaging is needed.